src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -152,8 +152,16 @@ boolean convertToCdf() {
   LoadedCvrData parseAndCountCastVoteRecords(BiConsumer<Double, Double> progressUpdate)
       throws CastVoteRecordGenericParseException {
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
+    boolean setUpLoggingSuccess = setUpLogging(config.getOutputDirectory(), "check-ballot-counts");
```

Comment:
Agree that it's probably useless, but it is outputted to the GUI log, so I don't know if there's any downside to having things in the GUI that's not in the audit log. (That already happens with some GUI logging messages, like the initial load.)

Reply:
Speaking with @yezr:
- Don't include Chehck Ballot Counts audit log
- Do check checksums to ensure no change between Check and Tabulate

I'll get to this!

Reply:
I'm looking into this and it's not as straightforward as I had hoped because some CVRs are directories, not files.

Some considerations:
1. I could hash every file in a directory, but that several downsides, including having to recursively search a potentially very big directory with unnecessary files. We don't mandate that the CVR directory is free of clutter today, and I don't know if we should add that restriction. What if the Dominion CVR directory is just C:/ ? It will also fail if files are opened and temp hidden auto-save files are created, or .DS_STORE on mac.
2. I could enumerate every file we read in the directory, but this feels error-prone: what if a future release reads a new file from the directory, and we forget to add it to the list of files to hash? That feels like an easy-to-miss bug.

(1) is easier, but (2) seems safer. Still, it makes me want to move away from file hashing, and towards something a little cleaner.

1. I could hash each CVR, but I run into the same issue as (2): what if, in the future, a new field is added? This feels like a hard-to-catch bug once again, requiring manually spotting it via a PR.
2. I could hash the entire RCTab CVR, since we have plans to test and mandate that this file contains all information needed to exactly reproduce the election results.

So I'm leaning towards the latter: a checksum validation of the RCTab CVR. The downside is: (1) that means the file can never include timestamps, and (2) we can't inform the user which of their CVR sources has changed.


Thoughts?

Reply:
What a can of worms! I think we should kick the can of worms down the road and break this out into a separate issue, just submitting this PR as-is with the counting logging removed.

Reply:
created #853 to address stricter CVR comparisons outside of this PR